pkg/transformer/kubernetes/kubernetes_test.go

```diff
@@ -1145,3 +1145,68 @@ func TestNamespaceGenerationBlank(t *testing.T) {
 		}
 	}
 }
+
+func TestKubernetes_CreateSecrets(t *testing.T) {
+	// hardcoded
+	komposeDefaultObject := kobject.KomposeObject{
+		ServiceConfigs: map[string]kobject.ServiceConfig{"app": newServiceConfig()},
+		Secrets: types.Secrets{
+			"config-ini": types.SecretConfig{
+				Name: "debug-config-ini",
+				File: "../../../docs/CNAME",
+			},
+		},
+	}
+
+	type fields struct {
+		Opt kobject.ConvertOptions
+	}
+	type args struct {
+		komposeObject kobject.KomposeObject
+	}
+
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    []*api.Secret
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
Im assuming we can remove this?

Reply:
added one more test to generate secrets from file and merged new additions from main